test/functional/test_framework/key.py

```diff
@@ -86,13 +105,13 @@ def is_x_coord(self, x):
         return jacobi_symbol(x_3 + self.a * x + self.b, self.p) != -1
 
     def lift_x(self, x):
-        """Given an X coordinate on the curve, return a corresponding affine point."""
+        """Given an X coordinate on the curve, return a corresponding affine point for which the y-coordinate is even."""
```

Comment:
inconsistent capitalization and hyphenation

Reply:
Fixed.